routes/web.php

```diff
@@ -84,20 +84,22 @@
 
     Route::post('/integrations', [IntegrationController::class, 'store']);
 
-    Route::delete('/integrations/{id}', [IntegrationController::class, 'destroy']);
-    Route::patch('/integrations/{id}', [IntegrationController::class, 'update']);
+    Route::group(['middleware' => 'can:access-integration,id'], static function () {
+        Route::delete('/integrations/{id}', [IntegrationController::class, 'destroy']);
+        Route::patch('/integrations/{id}', [IntegrationController::class, 'update']);
 
-    Route::patch('/integrations/{id}/urls', [IntegrationController::class, 'updateUrls']);
-    Route::post('/integrations/{id}/urls', [IntegrationController::class, 'storeUrl']);
-    Route::delete('/integrations/{id}/urls/{urlId}', [IntegrationController::class, 'destroyUrl']);
+        Route::patch('/integrations/{id}/urls', [IntegrationController::class, 'updateUrls']);
```

Comment:
Now I notice these are `PATCH` methods instead of `PUT` methods although all URLs are updated. I think it is maybe more consistent and evident to just use `PUT` everywhere.

Reply:
Yes the implementation in the frontend has changed. Now we only send the changed. So `PUT` does make sense indeed.

Reply:
There are some changes in Hande's settings PR. I'll take a look at the endpoints and there corresponding http verb there.